### ISSUE TITLE
Don't bundle compile output in .jar

### ISSUE THREAD

**Issue by thheller** [braintripping/re-view#8]
The `re-view` `0.4.6` jar contains a whole bunch of compiled files. The `public/` files should never be in there.

Its a side effect of compiling into `resources` which I do not recommend doing.

```
[zilence@zpro ~/.m2/repository/re-view/re-view/0.4.6]$ jar -xvf re-view-0.4.6.jar
 inflated: META-INF/MANIFEST.MF
 inflated: META-INF/maven/re-view/re-view/pom.xml
 inflated: META-INF/leiningen/re-view/re-view/project.clj
 inflated: project.clj
 inflated: META-INF/leiningen/re-view/re-view/README.md
  created: META-INF/
  created: META-INF/maven/
  created: META-INF/maven/re-view/
  created: META-INF/maven/re-view/re-view/
 inflated: META-INF/maven/re-view/re-view/pom.properties
  created: public/
 inflated: public/.DS_Store
  created: public/js/
 inflated: public/js/.DS_Store
  created: public/js/test/
  created: public/js/test/cljs/
  created: public/js/test/cljs/core/
 inflated: public/js/test/cljs/core/match.cljs
 inflated: public/js/test/cljs/core/match.cljs.cache.edn
 inflated: public/js/test/cljs/core/match.js
 inflated: public/js/test/cljs/core/match.js.map
 inflated: public/js/test/cljs/core.cljs
 inflated: public/js/test/cljs/core.js
 inflated: public/js/test/cljs/core.js.map
 inflated: public/js/test/cljs/pprint.cljs
 inflated: public/js/test/cljs/pprint.cljs.cache.edn
 inflated: public/js/test/cljs/pprint.js
 inflated: public/js/test/cljs/pprint.js.map
  created: public/js/test/cljs/spec/
 inflated: public/js/test/cljs/spec/alpha.cljs
 inflated: public/js/test/cljs/spec/alpha.cljs.cache.edn
 inflated: public/js/test/cljs/spec/alpha.js
 inflated: public/js/test/cljs/spec/alpha.js.map
  created: public/js/test/cljs/spec/gen/
 inflated: public/js/test/cljs/spec/gen/alpha.cljs
 inflated: public/js/test/cljs/spec/gen/alpha.cljs.cache.edn
 inflated: public/js/test/cljs/spec/gen/alpha.js
...
```

**Comment by mhuebert** [comment]
Thanks. I've released `0.4.8` which should have this stuff removed.

**Comment by thheller** [comment]
`re-db` is also guilty of this.
